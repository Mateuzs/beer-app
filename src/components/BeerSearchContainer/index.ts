export { default as BeerSearchContainer } from "./BeerSearchContainer";
