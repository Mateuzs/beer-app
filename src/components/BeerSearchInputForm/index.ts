export { default as BeerSearchInputForm } from "./BeerSearchInputForm";
