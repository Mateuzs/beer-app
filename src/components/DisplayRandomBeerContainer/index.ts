export { default as DisplayRandomBeerContainer } from "./DisplayRandomBeerContainer";
