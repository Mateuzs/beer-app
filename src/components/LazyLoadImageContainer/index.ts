export { default as LazyLoadImageContainer } from "./LazyLoadImageContainer";
