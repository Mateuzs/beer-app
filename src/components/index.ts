export { BeerSearchContainer } from "./BeerSearchContainer";
export { DisplayRandomBeerContainer } from "./DisplayRandomBeerContainer";
export { NavBar } from "./NavBar";
export { SearchButton } from "./SearchButton";
export { LazyLoadImageContainer } from "./LazyLoadImageContainer";
export { ErrorMessage } from "./ErrorMessage";
export { BeerSearchInputForm } from "./BeerSearchInputForm";
export { BeerList } from "./BeerList";
