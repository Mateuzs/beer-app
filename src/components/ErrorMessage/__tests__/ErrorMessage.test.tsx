// external
import React from "react";
import { shallow, mount } from "enzyme";
// components
import { ErrorMessage } from "../../../components";

describe("ErrorMessage component", () => {
  test("should match the snapshot and styles", () => {
    const mockedData = {
      errorMessage: "ERROR_MESSAGE",
    };
    const wrapper = shallow(<ErrorMessage {...mockedData} />);

    expect(wrapper).toMatchSnapshot();
  });

  test("should render proper error message", () => {
    const mockedData = {
      errorMessage: "ERROR_MESSAGE",
    };
    const component = mount(<ErrorMessage {...mockedData} />);

    expect(component.find('[data-test-id="error-message-info"]').text()).toEqual("ERROR_MESSAGE");
  });
});
