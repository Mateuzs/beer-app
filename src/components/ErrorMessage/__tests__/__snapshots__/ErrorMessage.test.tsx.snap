// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage component should match the snapshot and styles 1`] = `
<div
  className="error-message-container"
  data-test-id="error-message-info"
>
  <h3
    className="error-message"
  >
    ERROR_MESSAGE
  </h3>
</div>
`;
